server/src/main/java/org/apache/druid/server/http/SegmentListerResource.java

```diff
@@ -327,6 +330,24 @@ public void onFailure(Throwable th)
     asyncContext.setTimeout(timeout);
   }
 
+  @GET
+  @Path("/loadCapabilities")
```

Comment:
@adarshsanjeev this adds double slash to the new URL as the resource class has a trailing slash already: `/druid-internal/v1/segments/`. I see the change-requests also follows a similar pattern. I think it should be fine as Jetty will likely normalize the path, but it would be good to avoid double slashes.

Reply:
Oh, weird, it seems to have always been this way.
Thanks for catching this, @abhishekrb19 !
Fixed it in #17830